index.bs

```diff
@@ -681,9 +683,22 @@ The header's ABNF is:
   Sec-CH-UA-Platform-Version = sf-string
 ```
 
+The 'Sec-CH-UA-WoW64' Header Field {#sec-ch-ua-wow64}
```

Comment:
Do we want to have `Sec-CH-App-Bitness` instead that offers a generic hint, instead of windows specific?

Pros:
- It's not something windows-specific
- It could be used for other use cases (e.g. to disable some features on 32 bit binaries on 64 bit platforms)

Cons:
- If we'd rather not offer this as a generic hint, keeping it windows specific to just make the UA->UA-CH transition easier makes sense.  There are other ways for sites to know if they should enable/disable certain features that shouldn't just be tied to 32-bit app on 64-bit platform.

Reply:
> Do we want to have Sec-CH-App-Bitness instead that offers a generic hint, instead of windows specific?

I think if we had a handful of compelling use cases, I would prefer this. But since we don't, I opted for the safer "pave the backwards compatible cowpath" approach.

Reply:
Makes sense